### PR TITLE
Working with Node.js/javascript date and time objects using Moment.js date library

### PR DESCRIPTION
**Pull request by kimkimani**
Date and time Objects are those development difficulties that we jump into at some point when developing Node.js/Javascript applications. This article discusses how to handle these objects with Moment.js library.

### CODE REVIEW

**Comment by kimkimani** [comment]
Ready for review

**Comment by hectorkambow** [comment]
@botahamec pinging you on this article for a tech review - please check that the info is accurate and the code snippets work as intended. 🚀  
(leave any comments-if the authors needs to change anything) 
Feel free to add any comments to add any value to the article you see fit 
Thank you

**Review by botahamec — CHANGES_REQUESTED** [review]
Overall, this was a very good introduction to Moment.js. Just make sure that all of your information is accurate.

**Comment by kimkimani** [comment]
@botahamec Should be better now

File: articles/node-date-and-time-objects-with-moment/index.md [hunk 1]
```diff
@@ -0,0 +1,531 @@
...
+```
+
+[**Time to x**](https://momentjs.com/docs/#/displaying/to/)
+
+Displays dates other than now.
+
+```js
+const moment = require('moment');
+ const a = moment([2020, 4, 12]);
+ const a = moment([2020, 8, 13]);
```
Comment (botahamec) [inline articles/node-date-and-time-objects-with-moment/index.md:161]: You have two variables called `a`. I think this is supposed to be called `b`

File: articles/node-date-and-time-objects-with-moment/index.md [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+
+Eg1: check if a moment [is before](https://momentjs.com/docs/#/query/is-before/) other moments.
+
+```js
+const moment = require('moment');
+console.log(moment('2020-08-12').isBefore('2020-09-20'));
+console.log(moment('2020-08-12').isBefore('2020-07-11'));
+```
+
+Eg2 check if a moment [is same to](https://momentjs.com/docs/#/query/is-same/) another moment.
```
Comment (botahamec) [inline articles/node-date-and-time-objects-with-moment/index.md:347]: Instead of "is same to", use "is same as"

**Review by botahamec — CHANGES_REQUESTED** [review]
@kimkimani This is certainly better, but there are still a few problems. Don't be discouraged, but try to fix these, please.

File: articles/node-date-and-time-objects-with-moment/index.md (hunk 2 above)
Comment (kimkimani) [reply articles/node-date-and-time-objects-with-moment/index.md]: Thank you @botahamec

**Comment by kimkimani** [comment]
@botahamec changes addressed

**Comment by kimkimani** [comment]
@hectorkambow 
Tried to expound more on some "Query optimization" points

**Comment by botahamec** [comment]
@hectorkambow This is ready

**Comment by hectorkambow** [comment]
@botahamec thank you for the review and fantastic comments and suggestions 
@kimkimani please review those changes (suggested) you made - to better incorporate those types of edits before submitting - to cut down review time and the # of revisions 🚀

**Comment by hectorkambow** [comment]
@ninjaginja  could you give this "Moment.js date library" article a final tech review - to make sure we didn't miss anything when you have a moment. :D 
🚀  
I did a preliminary review - and had Mike do a tech review as well

**Pull request merged**